### PR TITLE
Markdown implementation

### PR DESCRIPTION
**Pull request by Ernesto905**

### CODE REVIEW

**Review by LifeBringer — APPROVED** [review]
Changes requested

File: src/contexts/home.jsx [hunk 1]
```diff
@@ -31,8 +39,12 @@ export const HomeContextProvider = ({ children }) => {
           setLoading(true)
           let json = await axios.get('https://api.thenewsapi.com/v1/news/top?api_token='+NEWS_API_KEY+"&search="+currentSport+"&language=en&sort=published_at&limit=2&categories=sports")
           setNews(json.data.data)
+          console.log(json)
```
Comment (LifeBringer) [inline src/contexts/home.jsx]: Needs to be deleted

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Console.log

File: src/contexts/home.jsx (hunk 1 above)
Comment (Rebecca-CYLi) [inline src/contexts/home.jsx]: console log?

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
1 more console to remove and you good

**Pull request merged**